ocp_resources/node_network_configuration_policy.py

```diff
@@ -326,6 +331,28 @@ def clean_up(self):
 
         super().clean_up()
 
+    def append(self, nncp, spec_to_append):
```

Comment:
Where `spec_to_append` is a list, maybe just `specs`, wdyt?

Reply:
This makes sense. I'll update this